metadata-jobs/mae-consumer-job/src/main/java/com/linkedin/metadata/kafka/MaeConsumerApplication.java

```diff
@@ -14,8 +14,10 @@
 @SpringBootApplication(exclude = {ElasticsearchRestClientAutoConfiguration.class, CassandraAutoConfiguration.class,
     SolrHealthContributorAutoConfiguration.class})
 @ComponentScan(basePackages = {
+        "com.linkedin.gms.factory.kafka",
```

Comment:
What were the implications of NOT having these?

Reply:
Exception when not enabling consumers.